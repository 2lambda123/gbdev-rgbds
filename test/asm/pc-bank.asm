SECTION "Fixed bank", ROMX,BANK[42]
X = BANK(@)

SECTION "Something else", OAM
Y = BANK("Fixed bank")

    PRINTT "@: {X}\nStr: {Y}\n"
