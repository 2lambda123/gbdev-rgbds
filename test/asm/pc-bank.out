ERROR: pc-bank.asm(2):
    Source address $2a00 not in $FF00 to $FFFF
ERROR: pc-bank.asm(11):
    @'s bank is not known yet
ERROR: pc-bank.asm(11):
    Non-constant expression
@: $2A
Str: $2A
